packages/webview_flutter/example/tizen/tizen-manifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest package="org.tizen.webview_flutter_tizen_example" version="1.0.0" api-version="5.5" xmlns="http://tizen.org/ns/packages">
     <profile name="common"/>
-    <ui-application appid="org.tizen.webview_flutter_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" taskmanage="true" nodisplay="false" api-version="6" launch_mode="single">
+    <ui-application appid="org.tizen.webview_flutter_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" taskmanage="true" nodisplay="false" api-version="4" launch_mode="single">
```

Comment:
Is the api-version="4" right? Actually, the api-version is different with manifest's one.

Reply:
This means the .Net API version, so all plugins example use version 4